lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold;
+  @include container-horizontal;
+  @include nx-truncate-ellipsis;
+  border: 1px solid;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  height: 32px;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-md 6px $nx-spacing-md;
+  // Uppercase Min so SCSS ignores it and uses CSS min, not SCSS min
+  max-width: Min(100%, 320px);
+}
+
+.nx-tag--selectable {
+  align-items: center;
+  cursor: pointer;
+  display: inline-flex;
+  padding-right: $nx-spacing-xs;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+    box-shadow: $nx-focus-box-shadow;
+    outline: none;
+  }
+
+  &:hover {
+    border-color: $nx-grey-900;
+  }
+}
+
+.nx-tag--selected {
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+  }
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+}
+
+.nx-tag__action {
+  margin-left: $nx-spacing-md;
+}
+
+// Available tag colours
+@mixin tag-color($tag-name, $tag-color-dark, $tag-color-light) {
+  .nx-tag--#{$tag-name} {
+    background-color: $tag-color-dark;
+    border-color: $tag-color-dark;
+  
+    &.nx-tag--unselected {
+      background-color: $tag-color-light;
+    }
+  }
+}
+
+@include tag-color(default, #bbc5f6, #e7eaf9);
```

Comment:
Why aren't these swatch colors?

Reply:
@dturner32 ^^

Reply:
because the Indigo palette was never added to zeplin, that's on me. I just added it and updated the color variables. I also added an implementation ticket in Jira.